app/src/main/java/com/nextcloud/talk/controllers/ConversationInfoController.java

```diff
@@ -161,15 +161,17 @@ protected void onViewBound(@NonNull View view) {
     }
 
     private void setupAdapter() {
-        if (adapter == null) {
+        if (adapter == null && getActivity() != null) {
```

Comment:
You could assign getActivity to a variable and check it only once in preceding if.
This might be a bit better to read:

```
Activity activity = getActivity();
if (activity != null) {
 	if (adapter == null) {}

	if (recyclerView != null) {}
}
```

Reply:
I thought about this. What if Activity disappears between the first place where I need it and the second one? :)

Unlikely, sure, but possible? :)

Reply:
As there are no time intensive operation in it, I think there is a very little chance to have this, but of course it could happen. 
I still would bet that you never see a crash regarding this ;-)

Reply:
It's not about it being time intensive, but the user might interrupt the app. Anyway, I do agree it's unlikely like I mentioned above. Will change it.

Reply:
Done post-merge!